packages/manager/modules/vps/src/dashboard/migrate/migrate.controller.js

```diff
@@ -8,23 +8,25 @@ export default class {
     coreConfig,
     ovhManagerProductOffersService,
     VpsHelperService,
+    vpsMigrateService,
   ) {
     this.atInternet = atInternet;
     this.coreConfig = coreConfig;
     this.ovhManagerProductOffersService = ovhManagerProductOffersService;
     this.VpsHelperService = VpsHelperService;
     this.MEMORY_MULTIPLE = MEMORY_MULTIPLE;
+    this.vpsMigrateService = vpsMigrateService;
```

Comment:
Probably useless. I don't locate an y usage.
Mistake ?

Reply:
removed it from the code.